src/Selenium2Library/keywords/element.py

```diff
@@ -171,10 +186,11 @@ def page_should_not_contain_element(self, locator, message='', loglevel='INFO'):
         Key attributes for arbitrary elements are `id` and `name`. See
         `introduction` for details about locating elements.
         """
-        self._page_should_not_contain_element(locator, None, message, loglevel)
-
-    # Public, attributes
+        self.page_not_contains_element(
+            locator, message=message, loglevel=loglevel
+        )
```

Comment:
This is the last time I mention this, but I don't think it is generally a good idea to move all arguments into separate row if having them in the same row as the method call makes the line `> 80`. It works nicely in some cases, especially if arguments themselves are long, but in general I consider it a lot better to split just the offending arguments to own row:

```python
self.page_not_contains_element(locator, message=message, 
                               loglevel=loglevel)
```